src/cargo/util/rustc.rs

```diff
@@ -14,9 +14,9 @@ impl Rustc {
     ///
     /// If successful this function returns a description of the compiler along
     /// with a list of its capabilities.
-    pub fn new<P: AsRef<Path>>(path: P) -> CargoResult<Rustc> {
-        let mut cmd = try!(util::process(path.as_ref()));
-        cmd.arg("-vV");
+    pub fn new<P: AsRef<Path>>(path: P, cwd: &Path) -> CargoResult<Rustc> {
```

Comment:
It should be fine to eschew this reference to `cwd` because it's not actually needed when executing the compiler (e.g. no files are being passed as input)